internal/backends/postgresql/collection.go

```diff
@@ -101,6 +101,11 @@ func (c *collection) Query(ctx context.Context, params *backends.QueryParams) (*
 		args = append(args, sortArgs...)
 	}
 
+	if params.Limit != 0 {
+		q += fmt.Sprintf(` LIMIT %s`, placeholder.Next())
```

Comment:
Please port that basic logic to the SQLite, too. We will not use it for basic queries with filter and sort, but we will need to for #3459